code/game/objects/items/storage/secure.dm

```diff
@@ -38,21 +39,21 @@
 				user.show_message("<span class='notice'>You [open ? "open" : "close"] the service panel.</span>", 1)
 			return
 		if ((istype(W, /obj/item/device/multitool)) && (src.open == 1)&& (!src.l_hacking))
-			user.show_message("<span class='danger'>Now attempting to reset internal memory, please hold.</span>", 1)
-			src.l_hacking = 1
-			if (do_after(usr, 100*W.toolspeed, target = src))
+			to_chat(user, text("<span class='danger'>Now attempting to reset internal memory, please hold.</span>")

```

Comment:
Blargh, no reason to use the `text()` wrapper here either! Just `user, "blah"` is fine.